package/MDAnalysis/coordinates/DCD.py

```diff
@@ -498,22 +498,23 @@ def _read_frame(self, frame):
         ts.frame = frame
         return ts
 
-    def timeseries(self, asel, start=0, stop=-1, skip=1, format='afc'):
+    def timeseries(self, asel, start=None, stop=None, step=None, format='afc'):
```

Comment:
You need to keep `skip` and deprecate it. Just changing the kwarg will break code. For a start, *introduce* `step` and let it behave in the documented, standard way. And then read on...

There's also an open issue #818 on *exactly* `skip`'s non-standard behavior. Please check that issue, too, and solve the problem in this context and with input from the folks on #818.